feconf.py

```diff
@@ -235,6 +237,11 @@ def get_empty_ratings():
     },
 }
 
+# Prefixes for activity ids to denote the type of activity. These values should
+# not be changed.
+ACTIVITY_ID_PREFIX_EXPLORATION = 'e:'
```

Comment:
What do you mean by "store the activity types in the backend model"?

I wanted to introduce a canonical way of referring to activities using a single string. Even if we used 2-tuples of strings, we will need to have a string somewhere in that to denote the activity type...